nebari/initialize.py

```diff
@@ -78,20 +78,20 @@ def default_environments():
             "name": "dask",
             "channels": ["conda-forge"],
             "dependencies": [
-                "python",
-                "ipykernel",
+                "python=3.10.8",
+                "ipykernel=6.21.0",
                 "ipywidgets==7.7.1",
                 f"nebari-dask =={nebari_dask_version}",
```

Comment:
I didn't change this line. It seemed like it should stay like this, but I can pin it to a set version if needed.

The value that is assigned here: `nebari_dask_version = set_nebari_dask_version()`, is `2023.1.1`, which is what was suggested in the issue.

Reply:
yes, this should stay as-is